### DISCUSSION TITLE
pyrebase4 or pyrebase

### DISCUSSION THREAD

**Discussion by Youssef3082004** [flet-dev/flet#4375] [Q&A]
hello everybody, at first I had flet project with Firebase, so I used Pyrebase lib to connect with Firebase 

so when I transformed the main file with this library it didn't transform to APK and at the same time didn't generate an error just loading without progress
so what is the solution to this problem? 
and I will upload the log file and requirements file 

python version is 3.13
flet version is 0.24.1
pyrebase4 is 4.8.0
requests is 2.32.3

**Comment by Youssef3082004** [comment]
[log.txt](https://github.com/user-attachments/files/17757870/log.txt)
[requirements.txt](https://github.com/user-attachments/files/17757872/requirements.txt)

**Comment by tanmay-bhatgare** [comment]
Is pyrebase4 a pure python lib(entirely written in python) or the native one? If it's a native lib you first need to convert it to pure package, i don't think it is that easy, still you could give it a try.

**Reply by Youssef3082004** [reply]
@tanmay-bhatgare 
thanks for replying, so yes pyrebase4 is native lib I think that I will use firebase_Admin instead, I think this will solve the problem

**Reply by Youssef3082004** [reply]
Have you tried firebase_Admin to convert your app to APK?

**Reply by tanmay-bhatgare** [reply]
@youssef969 no I didn't. I didn't find any need to use firebase I use flet as quick UI maker along with python. I mostly work with static apps and for dynamic ones I use react native.

**Reply by Youssef3082004** [reply]
oh okay thank you for your reply 
is attention me to things that I don't know about
